OAK-Marketplace-Emotion-Recognition/README.md

```diff
@@ -0,0 +1,60 @@
+This contains the code for **Emotion Recognition using OpenCV AI KIT and Modelplace**. For more information - visit [OpenCV.AI](https://www.opencv.ai/).
+
+# OAK Emotion Recognition 
+
+![](images/demo.gif)
+
+## Requirements
+- Install Python 3.7 or higher and run: ```python3 -m pip install -r requirements.txt```
+
+- Install [FFmpeg](https://ffmpeg.org/download.html)
+
+- Install [Emotion Recognition OAK model](https://github.com/opencv-ai/oak-model-samples/tree/main/emotion_recognition_retail)
```

Comment:
Wrong link. The link must lead to https://github.com/opencv-ai/opencv-blog/tree/as/oak-modelplace-emotion-recognition/OAK-Marketplace-Emotion-Recognition#installation-of-emotion-recognition-oak-package. 

Need to use this link - https://github.com/opencv-ai/opencv-blog/tree/main/OAK-Marketplace-Emotion-Recognition#installation-of-emotion-recognition-oak-package. Currently, doesn't work but will do when PR is merged.

Reply:
And this should be before ffmpg